benchmarks/docker/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script runs the dockerized benchmarking framework, which in
+# turn will pull Nighthawk and Envoy in via docker. 
+
+set -eo pipefail
+set +x
+set -u
+
+# The benchmark logs and artifacts will be dropped here
+OUTDIR="/home/oschaaf/code/envoy-perf-vscode/nighthawk/benchmarks/tmp/"
```

Comment:
These are hardcoded paths, I think I may have said something about `$USER` in another thread.